initdb.py

```diff
@@ -1317,6 +1316,7 @@ def populate_database(minimal=False):
     "ManifestLegacyImage",
     "Image",
     "ProxyCacheConfig",
+    "RepositorySize",
```

Comment:
Yes, since the table is going to have no content but still exist it has to be added to this list. We error out if any table has no content during testing.